src/main/resources/molgenis_ui.xml

```diff
@@ -2,12 +2,7 @@
 <molgenis label="ASE" name="ase">
 	<menu label="Home" name="main">
 		<plugin id="home" name="Home" url="home"/>
-		<plugin id="ase" name="ASE" url="dataexplorer?entity=ASE"/>
-        <plugin id="samples" name="Samples" url="dataexplorer?entity=Samples"/>
-        <menu label="Import" name ="import">
-			<plugin id="importwizard" name="Import" url="importwizard"/>
-			<plugin id="vcfimporter" name="VCF" url="vcfimporter"/>
-		</menu>
+		<plugin id="importwizard" name="Upload" url="importwizard"/>
```

Comment:
If I understood correctly, the menu cannot be changed through excel yet. You have to drop the runtimeproperty before you can import it. And if you've dropped it, you've also lost the menu. So the most efficient way to deploy it is probably through molgenis_ui.xml.